### DISCUSSION TITLE
在线等，之前设备找不到了，如何在新的设备上登录Pro账号？

### DISCUSSION THREAD

**Discussion by HawkeFia** [getlantern/lantern-forum-cn#1045] [Q&A]
我之前登录Pro账号的手机找不到了，在新的设备上登录，让我提供授权pin码，如何在新的设备上，不使用pin码登录使用Pro账号啊？国内的邮箱也收不到官方的验证码，着急！有知道的大佬吗？先谢了！

**Comment by MaxMKMax** [comment]
@HawkeFia https://lantern.io/zh-Hans/faq
您好. 希望这个链接可以帮到您.

**Comment by MaxMKMax** [comment]
@HawkeFia 如果您依然搜不到验证码. 那我另外帮您提交问题上去处理. 您需要提供我支付单据和账号邮箱.
